css/index.css

```diff
@@ -70,6 +70,28 @@ p {
     width: 800px;
     margin: 0 auto;
 }
+header {
+    margin: 20px 0px 30px 0px;
+}
+header nav {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    font-size: 14px;
+}
+nav a {
+    width: 15%;
+    padding: 0 2%;
+    text-decoration: none;
+    color: black;
+}
+nav a:hover {
+    transform: scale(1.3);
```

Comment:
**Awesome!!**
You have re-imagined CSS animations using just transition and transform properties!!